types/three/src/renderers/webxr/WebXRManager.d.ts

```diff
@@ -16,15 +26,24 @@ export class WebXRManager extends EventDispatcher {
      */
     isPresenting: boolean;
 
+    /**
+     * @default true
+     */
+    cameraAutoUpdate: boolean;
+
     getController(index: number): Group;
     getControllerGrip(index: number): Group;
     getHand(index: number): Group;
     setFramebufferScaleFactor(value: number): void;
     setReferenceSpaceType(value: XRReferenceSpaceType): void;
     getReferenceSpace(): XRReferenceSpace | null;
+    getBaseLayer(): XRWebGLLayer | any // XRProjectionLayer
+    getBinding(): any // XRWebGLBinding
```

Comment:
`XRWebGLBinding` is not defined even in `@types/webxr` :weary:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webxr/index.d.ts

Reply:
Indeed, their types are rather neglected atm. See my original comment, fixed links to Babylon's types which are more correct. Can possibly vendor those until we can get these fixed upstream.